main.py

```
@@ -7,6 +7,7 @@
 import requests
 from apscheduler.schedulers.blocking import BlockingScheduler
 from dotenv import load_dotenv
+from tinydb import Query, TinyDB
```

Ruff (0.8.2)

`tinydb.Query` imported but unused

Remove unused import: `tinydb.Query`

(F401)